scripts/script.js

```diff
@@ -0,0 +1,31 @@
+jQuery(
```

Comment:
Try now to achieve the same result without using jQuery at all :)
You can create similar repository without attaching jQuery lib.